x-pack/test/functional/services/ml/data_frame_analytics_creation.ts

```diff
@@ -124,13 +124,21 @@ export function MachineLearningDataFrameAnalyticsCreationProvider(
       await this.assertJobDescriptionValue(jobDescription);
     },
 
-    // async assertExcludedFieldsSelection(expectedSelection: string[]) {
-    //   const actualSelection = await comboBox.getComboBoxSelectedOptions(
-    //     'mlAnalyticsCreateJobWizardExcludesSelect'
-    //   );
+    async assertSourceDataPreviewExists() {
+      await testSubjects.existOrFail('mlAnalyticsCreationDataGrid loaded', { timeout: 5000 });
+    },
+
+    async assertIncludeFieldsSelectionExists() {
+      await testSubjects.existOrFail('mlAnalyticsCreateJobWizardIncludesSelect', { timeout: 5000 });
+    },
+
+    // async assertIncludedFieldsSelection(expectedSelection: string[]) {
```

Comment:
Can this section be uncommented yet? If not, worth adding a comment explaining what they are waiting on.

Reply:
This will be added back in but I wanted to work with @pheyos since I don't have much context on how to see which checkboxes are checked and thought it would be more efficient to chat with him about it.